.devcontainer/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.20-bookworm as builder
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y clang-15 libelf-dev \
+        libbpf-dev gcc binutils \
```

Comment:
is libbpf-dev needed? we have it as submodule now

Reply:
fixed